toolchain/sem_ir/formatter.cpp

```diff
@@ -456,27 +452,34 @@ class FormatterImpl {
       out_ << ": ";
       FormatType(sem_ir_.insts().Get(param_id).type_id());
     }
+
+    out_ << (is_implicit ? "]" : ")");
   }
 
+  // Prints instructions for a code block.
   auto FormatCodeBlock(InstBlockId block_id) -> void {
     if (block_id.is_valid()) {
       FormatCodeBlock(sem_ir_.inst_blocks().Get(block_id));
     }
   }
 
+  // Prints instructions for a code block.
   auto FormatCodeBlock(llvm::ArrayRef<InstId> block) -> void {
     for (const InstId inst_id : block) {
       FormatInst(inst_id);
     }
   }
 
+  // Prints a code block with braces, intended to be used trailing after other
+  // content on the same line (may wrap lines).
```

Comment:
What do you mean by "may wrap lines"? I think of that as meaning inserting line breaks to try to maintain a particular line width, but I didn't think the formatter ever did that.

Reply:
Just, may add line breaks.